datameta/api/metadatasets.py

```diff
@@ -47,11 +49,12 @@ class MetaDataSetResponse(DataHolderBase):
     submission_id        : Optional[str] = None
     service_executions   : Optional[Dict[str, Optional[MetaDataSetServiceExecution]]] = None
 
-def render_record_values(metadata:Dict[str, MetaDatum], record:dict) -> dict:
+
+def render_record_values(metadata: Dict[str, MetaDatum], record: dict) -> dict:
     """Renders values of a metadataset record. Please note: the record should already have passed validation."""
     record_rendered = record.copy()
     for field in metadata:
-        if not field in record_rendered.keys():
+        if field not in record_rendered.keys():
             # if field is not contained in record, add it as None to the record:
             record_rendered[field] = None
```

Comment:
```suggestion
        # if field is not contained in record, add it as None to the record:
        record_rendered.setdefault(field, None)        
```

Reply:
Nice! How did I not know about that one!

Reply:
Didn't work in the end, since after line 59 still comes a `continue` statement dependent on the same condition. But still nice to know, may prove useful in the future.